test-poc/examples/src/test/java/org/keycloak/test/examples/WelcomePageTest.java

```diff
@@ -3,14 +3,13 @@
 import org.junit.jupiter.api.Test;
 import org.keycloak.test.framework.annotations.KeycloakIntegrationTest;
 import org.keycloak.test.framework.page.WelcomePage;
-import org.keycloak.test.framework.annotations.TestWebDriver;
-import org.openqa.selenium.WebDriver;
+import org.keycloak.test.framework.annotations.WebDriver;
 
 @KeycloakIntegrationTest(config = NoAdminUserKeycloakTestServerConfig.class)
 public class WelcomePageTest {
 
-    @TestWebDriver
-    WebDriver driver;
+    @WebDriver
+    org.openqa.selenium.WebDriver driver;
```

Comment:
So we have a little conflict. But I think this is fine

Reply:
Can we think of a better name for this case? I'm also not convinced with the conflict

Reply:
Maybe `WebBrowser`, `Browser`, `BrowserDriver`, or just `Driver`? :car:

Reply:
What usually did was name it DriverHelper